src/decorators/parameter.ts

```diff
@@ -33,7 +33,7 @@ export function Request(): Function {
  *
  * @param {string} [name] The name of the path parameter
  */
-export function Path(name?: string): Function {
+export function Path(name?: string, example?: string): Function {
```

Comment:
`string` is not accurate either since path variables can be numbers, date, bool, etc (I believe)